### PR TITLE
magento/devdocs#: Added rowbuilder info

### PR DESCRIPTION
**Pull request by DashukVolodymyr**
## Purpose of this pull request

This pull request (PR) describes changing in RowBuilder widget doc.

## Affected DevDocs pages

-  https://devdocs.magento.com/guides/v2.3/javascript-dev-guide/widgets/widget-row-builder.html

### CODE REVIEW

**Review by BarnyShergold — APPROVED** [review]
For some reason the linting tests failed but I cannot see why

File: src/guides/v2.3/javascript-dev-guide/widgets/widget-row-builder.md [hunk 1]
```diff
@@ -54,6 +54,50 @@ The following example shows a PHTML file using the script:
...
+            "btnRemoveSelector": ".action.remove",
+            "formDataPost": data
+        });
+    });
+</script>
+```
+
+This is provided during widget instantiation. For example:
+
+```
```
Comment (atwixfirster) [inline src/guides/v2.3/javascript-dev-guide/widgets/widget-row-builder.md:93]: @DashukVolodymyr , could you please add `text` after ```

File: src/guides/v2.3/javascript-dev-guide/widgets/widget-row-builder.md [hunk 2]
```diff
@@ -54,6 +54,50 @@ The following example shows a PHTML file using the script:
...
+<script>
+    require([
+        'jquery',
+        'rowBuilder'
+    ], function ($) {
+        'use strict';
+
+        var data = {
+            formData: [
+                ["field1-name1", "field2-name1", "field3-name1"],
```
Comment (atwixfirster) [inline src/guides/v2.3/javascript-dev-guide/widgets/widget-row-builder.md:69]: Also it's good to see for the readers:
- sample of HTML code for `row-container`
- sample of HTML code for `row-template`

@DashukVolodymyr , could you please provide them?

Thank you!

File: src/guides/v2.3/javascript-dev-guide/widgets/widget-row-builder.md (hunk 1 above)
Comment (DashukVolodymyr) [reply src/guides/v2.3/javascript-dev-guide/widgets/widget-row-builder.md]: Hi @atwixfirster,  @BarnyShergold 
Thanks for your notes.

I think for this reason the linting tests did not pass!
I added `text` after ``` as requested.

Please proceed.
Thank you!

File: src/guides/v2.3/javascript-dev-guide/widgets/widget-row-builder.md (hunk 2 above)
Comment (DashukVolodymyr) [reply src/guides/v2.3/javascript-dev-guide/widgets/widget-row-builder.md:113]: @atwix-integrations 
I added HTML code for `row-container` and `row-template`.
Could you please proceed with the code review?

Thank you!

Comment (atwixfirster) [reply src/guides/v2.3/javascript-dev-guide/widgets/widget-row-builder.md:113]: Looks ok to me. ✅ 

Thank you!

File: src/guides/v2.3/javascript-dev-guide/widgets/widget-row-builder.md (hunk 1 above)
Comment (atwixfirster) [reply src/guides/v2.3/javascript-dev-guide/widgets/widget-row-builder.md]: Looks ok to me. ✅ 

Thank you!

**Comment by atwixfirster** [comment]
@dobooth , the PR is approved by @BarnyShergold and ready for your review 😊 

Thank you!

**Review by dobooth — APPROVED** [review]

**Comment by dobooth** [comment]
running tests

**Pull request merged**